apps/communications/contacts/views/details/js/boot.js

```diff
@@ -0,0 +1,58 @@
+/* global DetailsUI, Details, LazyLoader, fbLoader, ContactsService */
+'use strict';
+
+window.addEventListener('DOMContentLoaded', function() {
+  LazyLoader.load([
+    '/shared/js/l10n.js'], function() {
+    LazyLoader.load([
+      document.getElementById('view-contact-details')
```

Comment:
Once that we have this, I would like to measure the performance of doing this or directly having the template loaded into the html already.